src/did/src/block.rs

```diff
@@ -794,7 +800,7 @@ mod test {
                 &U256::new(10_u64.into()), // gas target 5
                 &base_fee
             ),
-            base_fee
+            U256::from(98u64)
```

Comment:
I may missed something, what is this 98? Can we add a comment to it?

Reply:
This is actually `100 - 0.125 * ((5-4) / 5) * 100`

Reply:
I've added a corresponding comment to the code